mathesar/tests/views/api/test_table_api.py

```diff
@@ -120,6 +121,34 @@ def test_table_detail(create_table, client):
     check_table_response(response_table, table, table_name)
 
 
+def test_table_type_suggestion(client, test_db_name):
+    table_name = 'Type Inference Table'
+    file = 'mathesar/tests/data/type_inference.csv'
+    schema = 'type_inference'
+    EXPECTED_TYPES = {
+        'col_1': 'numeric',
```

Comment:
The way types are represented here should match the way they're represented in the current table list/detail API. We use "INTEGER", "VARCHAR", etc.